### ISSUE TITLE
GZCGQ01LM update issue

### ISSUE THREAD

**Issue by jjnh** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#597]
Yesterday I received a Aqara GZCGQ01LM light sensor. Installation and linking to zigbee2mqtt went fine. The sensor showed up and was paired all devices in Domoticz were created correctly.

However one day later I notice the values are not updating in Domoticz. In the zigbee2mqtt plugin page I see that date and time of the sensor are updated, but in Domoticz all values stay on the last date after pairing.

I already removed it from the plugin, removed all the devices from Domoticz and re-paired it. Again same behavior. When I click the button on the side of the sensor. The last seen time updates, but Domoticz values and time stay the same.

There are also no updates visible in the Domoticz log for this sensor.

Does anyone have this same behavior?

**Comment by jjnh** [comment]
Impatient as I am :) experiments continued. I found a website that describes a process to keep the button on the sensor pressed for more than 10 seconds resetting it completely. This is what I did. After that I removed the sensor from Domoticz and from Zigbee2MQTT. Restarted the pairing and reconnected the device.

Now it has been working for half a day. Let's see if it still will be working in a few days.

One thing is odd though. I don't see sensor updates coming by in the Domoticz log anymore. Logging is set to True. Has this been changed in a recent release?
